### ISSUE TITLE
.sbv (Youtube) support

### ISSUE THREAD

**Issue by oytuntez** [captioning/captioning#9]
.sbv is starting to gain traction among Youtube ecosystem. It would be nice to support it. We will try to provide the implementation from what we use in MotaWord.

But I opened an issue anyway, in case somebody else can handle this faster than me.

**Comment by delphiki** [comment]
I wasn't aware of this format, it could be interesting indeed. :)

**Comment by delphiki** [comment]
The format seems pretty simple to implement (close to subrip), it should be hard to add it.

**Comment by delphiki** [comment]
Aaaaaaaand, I made a typo in the commit message :)

**Comment by oytuntez** [comment]
Thanks @delphiki! That was fast :) We will do tests on our side and if everything is fine, we'll start using your package for SBV as well.

**Comment by delphiki** [comment]
No problem, and feedback is always appreciated! :)

**Comment by oytuntez** [comment]
I tested this implementation, seems to be working pretty well as much as I could cover.

Thanks again!

**Comment by delphiki** [comment]
Good to hear!

**Comment by oytuntez** [comment]
@delphiki, weirdly the last cue of the file below is not processed:

```
0:05:40.000,0:05:46.000
Don’t think that you can just ignore them 
because they’re not your children or relatives.

0:05:46.000,0:05:51.000
Because every child in our society is 
a part of that society.

0:05:51.000,0:05:55.000
We should give them opportunities 
and only good things,…

0:05:55.000,0:05:59.000
…so that they can grow up to be 
good adults in the future.
```


Tested with different content, last sections are never processed. Any ideas?

**Comment by delphiki** [comment]
@oytuntez There was a bug in the SBV parser, this has been fixed in [version 2.5.2](https://github.com/captioning/captioning/releases/tag/2.5.2) :wink:

**Comment by oytuntez** [comment]
Thanks @delphiki! Will pull ASAP.
